kernel/src/cpu/cpuid.rs

```diff
@@ -39,7 +40,7 @@ pub unsafe fn copy_cpuid_table_to(dst: VirtAddr) {
         start.write_bytes(0, PAGE_SIZE);
         start
             .cast::<SnpCpuidTable>()
-            .copy_from_nonoverlapping(&*CPUID_PAGE, 1);
+            .copy_from_nonoverlapping(ptr::from_ref(*CPUID_PAGE), 1);
```

Comment:
Nit: you can simply do the following, as `ImmutAfterInitCell` implements `Deref`, and `&T` can be automatically coerced to `*const T` by the compiler:
```suggestion
            .copy_from_nonoverlapping(*CPUID_PAGE, 1);
```

Reply:
I thought I had tried that and hit an error, but apparently not.  I agree that this is better.